app/packages/components/src/scrollable.module.css

```diff
@@ -2,7 +2,7 @@
   scrollbar-color: var(--fo-palette-text-tertiary)
     var(--fo-palette-background-body);
 
-  scrollbar-gutter: stable;
+  scrollbar-gutter: auto;
```

Comment:
@benjaminpkane would this have any negative side-effect?  It removes scrollbar placeholder when container is not overflowing

Reply:
The sidebar scrollbar is set to `scroll` so I think it's ok